catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -407,6 +407,12 @@
         </bundle>
     </feature>
 
+    <feature name="catalog-core-validationfilterplugin" version="${project.version}"
```

Comment:
❓ Why are you adding the feature here specifically when the bundle is already started as one of the `catalog-core-plugin` here - https://github.com/codice/ddf/blob/e86ca0d388b820e5033768db13186d3312d343e3/catalog/catalog-app/src/main/resources/features.xml#L90

Reply:
ah right i modified that to reference the feature now.

Reply:
I still don't see the advantage, or why it should be started differently from the other plugins. Is the intent to make it such that downstream projects could skip the startup of `catalog-core-plugins` but start this plugin? Shouldn't we then wrap all the plugins as single-bundle features so they too could be independently managed?

Reply:
yes currently GSR doesn't start the `catalog-core-plugins`. Yes perhaps we need to break all the features up but that could have downstream effect. Or perhaps we just break the `catalog-core-validationfilterplugin` to be independently managed?

Reply:
I want to see consistency here. I'm a little surprised that your downstream project doesn't start _any_ of these other plugins; if it does, then I'd imagine it's either defined its own feature that starts the plugin bundles it needs (in which case you can just add this new plugin there) or it's defined features to contain each of the bundles individually.

Regardless, this inconsistency here in DDF is bothersome.

Reply:
Are you going to address this? It seems premature to trigger a CI build.

Reply:
you are right. i am closing this PR w/o merging. `catalog-core-plugins` feature is included by `catalog-app` which is included in `gsr-profile`, so `catalog-core-validationfilterplugin` won't be it own feature but declared as a bundle within the `catalog-core-plugins1